libmscore/ottava.cpp

```diff
@@ -184,7 +184,6 @@ Ottava::Ottava(const Ottava& o)
       _ottavaType = o._ottavaType;
       _numbersOnly = score()->styleB(StyleIdx::ottavaNumbersOnly);
       numbersOnlyStyle = o.numbersOnlyStyle;
-      lineWidthStyle = o.lineWidthStyle;
```

Comment:
Not sure this is the right thing, might be better to do all these initialization, but on writing the score to check whether the to be written setting is the default, and in that case do not write it

Reply:
I see what you mean, but I do not yet understand the concept of PropertyStyles completely. So it is a lot of trial and error for me. Yesterday I put a question regarding this in the mailing list (http://dev-list.musescore.org/), but it seems it is not yet published there. Is the list still in use?

Reply:
The list is in use and @wschweer answered you. You didn't get mail? I will check why it doesn't appear on nabble.

Reply:
See https://sourceforge.net/p/mscore/mailman/message/36186163/

Reply:
Yes, the list is still in use. I think... seems somehow the link to https://sourceforge.net/p/mscore/mailman/mscore-developer broke?
There I can see a question of yours, along with an reply from Werner

Reply:
Tha last update on http://dev-list.musescore.org/ is from July 26th. Was then when the new site went live?